### DISCUSSION TITLE
how to view sideways?

### DISCUSSION THREAD

**Discussion by goyalyashpal** [helix-editor/helix#5794] [Q&A]
- i was in some file i guess it was a typesetting one - hence, it had extremely long lines. 
- i had made some selection, then applied search on it.
- now, there came a situation where i needed to view the latter part of the line...

But i couldn't due to lack of soft-wrap, and as i would loose the selection if move cursor to right.

1. so, question is, how to move the view sideways (left/right)?
2. and while at it, after <strike>2520</strike> #2134 will be solved, is there any other way for doing `zjzjzjzjzj` as the popup appearing makes it jittery?

though i know that soft-wrap is being baked, but still, it would be nice to have view sideways too i guess - for those _going to be_ rare occassions where someone will not be using soft-wrap.

Current View options `z` are:

<img src="https://user-images.githubusercontent.com/19423063/216454872-b6f74d5e-1f53-46e0-838a-07a6561c5aae.png" width="470px"/>

**Comment by gabydd** [comment]
Two things there is now softwrap if you build from master, and you can use capital ``Z`` to make view mode "sticky" and then just continuously press `j`

**Reply by goyalyashpal** [reply]
thanks for sharing how to make view mode sticky. nice :)

Yes, i was aware about soft-wrap 😇

**Reply by goyalyashpal** [reply]
so, why doesn't view mode has `h` or `l` ??

that with sticky view `Z` would be all i need 🤔
